out.txt

```diff
@@ -0,0 +1,46 @@
+   Compiling peaq-pallet-rbac v0.1.0 (/Users/talhadar/work/peaq-pallet-rbac/pallet)
```

Comment:
Is this file added unexpected?

Reply:
I remember resetting that, but lemme have another look.

Reply:
Nice catch, have removed the file 🙌